### DISCUSSION TITLE
I can't run Pyansys-math: enter manually the path

### DISCUSSION THREAD

**Discussion by MayconSilva7** [ansys/pyansys-math#128] [Q&A] [answered]
Hello, 

I have a little experience with Python and I am new user of Ansys Python. First of all, congratulations for this. 

Well, I have been using the both on Windows, and as we know, it is common that there are problems with path. So, I attached the following image that shows the error I've been getting.

Thank you very much in advance. 

![error](https://github.com/ansys/pyansys-math/assets/122628753/63cdbb75-fa6a-4f98-8111-dc48dfba431e)

**Comment by clatapie** [comment]
Hello @MayconSilva7, 
Thank you for your interest in the PyAnsys project and for opening this discussion!
Could you please provide me the code that raises this issue?

**Comment by MayconSilva7** [comment]
Hey, 

I am just testing the Pyansys-math, so I write the following script:
```
import ansys.math.core.math as pymath

mm = pymath.AnsMath()

u = mm.ones(5)
v = mm.rand(5)
w = u + v

print(w)
```

I mean, it is something related to the path. I've been using the VSCode. And I added the folder to my working directory. 

Thank you very much again.

**Reply by clatapie** [reply, marked as answer]
As MAPDL is used in the background, it is necessary to have a licensing for that and localize its executable file. This [discussion ](https://github.com/ansys/pymapdl/discussions/1288) will probably helps you to find out were your ANSYS executable is located.
Once PyMAPDL is correctly set up, you can launch PyAnsys Math. The final code will probably look something like this:

```
from ansys.mapdl.core import launch_mapdl
import ansys.math.core.math as pymath

new_path = "C:\\Program Files\\ANSYS Inc\\v212\\ANSYS\\bin\\winx64\\ansys222.exe"
mapdl = launch_mapdl(exec_file=new_path)
mm = pymath.AnsMath(mapdl)
```
or, like this:

```
from ansys.mapdl import core as pymapdl
import ansys.math.core.math as pymath

new_path = "C:\\Program Files\\ANSYS Inc\\v212\\ANSYS\\bin\\winx64\\ansys222.exe"
pymapdl.change_default_ansys_path(new_path)

mm = pymath.AnsMath()
```

Let me know if it fixes the issue or if the problem persists!
